packages/playwright/src/runner/vcs.ts

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright Microsoft Corporation. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import childProcess from 'child_process';
+import { affectedTestFiles } from '../transform/compilationCache';
+import path from 'path';
+
+export async function detectChangedFiles(baseCommit: string): Promise<Set<string>> {
+  function gitFileList(command: string) {
+    try {
+      return childProcess.execSync(
```

Comment:
I think we want to pass `config.configDir` as `cwd` here, wdyt?

Reply:
smart!